lib/src/airship_flutter.dart

```diff
@@ -96,7 +96,7 @@ class NotificationResponseEvent {
 
 class PushReceivedEvent {
   final Map<String, dynamic>? payload;
-  final Notification notification;
+  final Notification? notification;
```

Comment:
The `_fromJson` method will still throw an error with this as optional because the `late` initializer on line 106 requires notification to be set before return even if it's nullable

Reply:
That's right, thank you